ngrinder-controller/src/main/java/org/ngrinder/common/service/ModelAspect.java

```diff
@@ -68,16 +68,18 @@ public void beforeSave(JoinPoint joinPoint) {
 				BaseModel<?> model = (BaseModel<?>) object;
 				Date lastModifiedDate = new Date();
 				model.setLastModifiedDate(lastModifiedDate);
+
 				User currentUser = userContext.getCurrentUser();
-				Long currentUserId = currentUser.getId();
+				long currentUserId = currentUser.getId();
 
 				model.setLastModifiedUser(userRepository.findOne(idEqual(currentUserId))
 					.orElseThrow(() -> new IllegalArgumentException("No user found with id : " + currentUserId)));
 
 				if (!model.exist() || model.getCreatedUser() == null) {
+					long factualUserId = currentUser.getFactualUser().getId();
```

Comment:
`getFactualUser()` always return not null value.

https://github.com/naver/ngrinder/blob/develop/ngrinder-core/src/main/java/org/ngrinder/model/User.java#L334